apps/mq-bridge-app/crates/cli/tests/docker/run_docker_tests.sh

```
@@ -0,0 +1,89 @@
+# in tests/integration/docker-compose -- the same files, images and ports the
+# engine's own integration tests use, so there is one definition per service.
+#
+# A service that is already up is reused and left running: the CLI tests then
+# cost only their own runtime. Anything this script started, it stops again.
+#
+#   ./run_docker_tests.sh              # the default set, under a minute cold
+#   ./run_docker_tests.sh all          # every backend
+#   ./run_docker_tests.sh kafka nats   # only these
+set -uo pipefail
+
+cd "$(dirname "$0")"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)